config.sh

```diff
@@ -63,10 +63,19 @@ function build_proj {
     if [ -e proj-stamp ]; then return; fi
     fetch_unpack https://download.osgeo.org/proj/proj-${PROJ_VERSION}.tar.gz
     suppress build_curl_ssl
+    get_modern_cmake
     (cd proj-${PROJ_VERSION:0:5}\
-        && ./configure --prefix=$PROJ_DIR --with-curl=$BUILD_PREFIX/bin/curl-config \
-        && make -j4 \
-        && make install)
+        && cmake . \
+        -DCMAKE_INSTALL_PREFIX=$PROJ_DIR \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_IPO=ON \
+        -DBUILD_APPS:BOOL=OFF \
+        -DBUILD_GIE:BOOL=OFF \
+        -DBUILD_TESTING:BOOL=OFF \
+        -DCMAKE_PREFIX_PATH=$BUILD_PREFIX \
```

Comment:
an option `CMAKE_INSTALL_LIBDIR=lib` may also need to be specified; see https://github.com/pygeos/pygeos/pull/416#issuecomment-955168761

Reply:
The build passed, but probably won't hurt to add it. Thanks :+1: